src/einsteinpy/plotting/geodesics_static.py

```diff
@@ -11,16 +13,67 @@
 
 class StaticGeodesicPlotter:
     """
-    Class for plotting static matplotlib plots
+    Class for plotting static matplotlib plots.
+
+    These are the features for making plots look good. 
+
+    Manual_Scaling :
+        If the user provides the attractor_radius_scale, then
+        the autoscaling will not work. This is checked by initialising the 
+        attractor_radius_scale by -1 and if the user enters the value then it 
+        will be >0 so the value won't remain -1 which is easily checked.
+    Auto Scaling : 
+        If the user does not provide the attractor_radius_scale, the value will be
+        initialised to -1 and then we will call the auto scaling function.
+        In autoscaling, the attractor radius is first initialised to the minimum distance between the
+        attractor and the object moving around it. Now, if this radius is greater than the 1/12th of minimum of
+        range of X and Y coordinates then, the radius is initialised to this minimum. This is done so that the plots
+        are easy to look at.
+    Color options : 
+        User can give the color to attractor of his/her choice.
+
+    Parameters
+    ----------
+    attractor_radius_scale :
+        Scales the attractor radius by the 
+        value given. Default is 1. It is used to 
+        make plots look more clear if needed.
+    attractor_color : 
+        Color which is used to denote the attractor.
+        Defaults to #ffcc00.
+
+    Functions and Class Variables
+    -----------------------------
+    get_swcradius : 
+        returns the schwarzschild radius of given mass
+	
+    __get_x_y : 
+        returns the X and Y coordinates of the particle moving around the attractor
+        This is given so the user can inspect the coordinates in the plot if needed.
+
+    get_curr_plot_radius : 
+        returns the current radius of the attractor
     """
 
-    def __init__(self, mass, time=0 * u.s, ax=None):
+    def __init__(
+        self,
+        mass,
+        time=0 * u.s,
+        ax=None,
+        attractor_radius_scale=-1.0,
+        attractor_color="#ffcc00",
+    ):
         self.ax = ax
         if not self.ax:
             _, self.ax = plt.subplots(figsize=(6, 6))
         self.time = time
         self.mass = mass
         self._attractor_present = False
+        self.attractor_color = attractor_color
+        self.attractor_radius_scale = attractor_radius_scale
+        self.xarr = False
```

Comment:
This should be a list initially?

Reply:
Also private

Reply:
Yes, it should be a list but datatypes can be automatically changed in python so I don't think it should be a problem, I will initialize it to empty list anyway.

Reply:
Its not a problem, but the code should not be misleading

Reply:
Done, initialised to empty numpy array